jetty-server/src/main/java/org/eclipse/jetty/server/CustomRequestLog.java

```diff
@@ -48,223 +47,274 @@
 import static java.lang.invoke.MethodType.methodType;
 
 /**
- * A flexible RequestLog, which produces log strings in a customizable format.
- * The Logger takes a format string where request characteristics can be added using "%" format codes which are
- * replaced by the corresponding value in the log output.
- * <p>
- * The terms server, client, local and remote are used to refer to the different addresses and ports
+ * <p>A flexible RequestLog, which produces log strings in a customizable format.</p>
+ * <p>The Logger takes a format string where request characteristics can be added using "%" format codes which are
+ * replaced by the corresponding value in the log output.</p>
+ * <p>The terms server, client, local and remote are used to refer to the different addresses and ports
  * which can be logged. Server and client refer to the logical addresses which can be modified in the request
  * headers. Where local and remote refer to the physical addresses which may be a proxy between the
- * end-user and the server.
+ * end-user and the server.</p>
  *
+ * <p>Percent codes are specified in the format <code>%MODIFIERS{PARAM}CODE</code> as follows:</p>
+ * <dl>
+ * <dt>MODIFIERS</dt>
+ * <dd>Optional list of comma separated HTTP status codes which may be preceded by a single "!" to indicate
+ * negation. If the status code is not in the list the literal string "-" will be logged instead of
+ * the resulting value from the percent code.</dd>
+ * <dt>{PARAM}</dt>
+ * <dd>Parameter string which may be optional depending on the percent code used.</dd>
+ * <dt>CODE</dt>
+ * <dd>A one or two character code specified by the table of format codes below.</dd>
+ * </dl>
  *
- * <br><br>Percent codes are specified in the format %MODIFIERS{PARAM}CODE
- * <pre>
- * MODIFIERS:
- *     Optional list of comma separated HTTP status codes which may be preceded by a single "!" to indicate
- *     negation. If the status code is not in the list the literal string "-" will be logged instead of
- *     the resulting value from the percent code.
- * {PARAM}:
- *     Parameter string which may be optional depending on the percent code used.
- * CODE:
- *     A one or two character code specified by the {@link CustomRequestLog} table of format codes.
- * </pre>
- *
+ * <!-- tag::documentation[] -->
```

Comment:
Why not include above documentation as well. It has some information which is not described in the documentation like the `MODIFIERS` section of the percent code.

Reply:
Done.